libs/sdk-ui-dashboard/src/presentation/drill/DrillConfigPanel/DrillToUrl/CustomUrlSection.tsx

```diff
@@ -0,0 +1,67 @@
+// (C) 2020-2022 GoodData Corporation
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+import { Button } from "@gooddata/sdk-ui-kit";
+
+import { DropdownSectionHeader } from "./DropdownSectionHeader";
+import { isDrillToCustomUrlConfig, UrlDrillTarget } from "../../types";
+
+type ToggleModalCallback = () => void;
+type CloseDropdownCallback = (e: React.MouseEvent) => void;
+
+interface CustomUrlSectionProps {
+    toggleModal: ToggleModalCallback;
+    closeDropdown: CloseDropdownCallback;
+    urlDrillTarget: UrlDrillTarget;
+}
+
+const UrlDropdownOption = ({
+    urlValue,
+    closeDropdown,
+}: {
+    urlValue: string;
+    closeDropdown: CloseDropdownCallback;
+}) => (
+    <div
+        onClick={closeDropdown}
+        className="gd-list-item gd-menu-item gd-drill-to-custom-url-option s-drill-to-custom-url-option gd-icon-hyperlink-disabled is-selected"
+    >
+        <span>{urlValue.length > 50 ? `${urlValue.substring(0, 50)}...` : urlValue}</span>
+    </div>
+);
+
+const EditButton = ({ urlValue, toggleModal }: { urlValue?: string; toggleModal: ToggleModalCallback }) => {
+    const intl = useIntl();
+    const buttonTitle = intl.formatMessage({
+        id: urlValue
```

Comment:
This will not be picked up by the i18n validation, please move the condition outside of the intl.formatMessage call